tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1556,6 +1556,23 @@ int tester_padv_create_sync(struct bt_le_per_adv_sync_param *create_params)
 	return err;
 }
 
+int tester_gap_padv_stop_sync(void)
+{
+	int err;
+
+	if (pa_sync == NULL) {
+		return -EBUSY;
```

Comment:
```suggestion
		return -EALREADY;
```

It's not really busy if it is not doing anything ;)